### PR TITLE
Chore: Add migration task

### PR DESCRIPTION
**Pull request by aocenas**
Moved code from https://github.com/grafana/grafana/pull/16175. In the former PR the migration was eventually removed, but this task seems like it could be useful.

### CODE REVIEW

File: pkg/services/sqlstore/migrator/migrator.go [hunk 1]
```diff
@@ -112,6 +117,7 @@ func (mg *Migrator) Start() error {
 		}
 	}
 
+	mg.Logger.Info(fmt.Sprintf("Executed %v migrations", migrationsCount))
```
Comment (aocenas) [inline pkg/services/sqlstore/migrator/migrator.go]: Makes sure it is easy to spot if no migration was run.

**Review by marefr — COMMENTED** [review]
Looks good :+1: 

Should we document this command? http://docs.grafana.org/administration/cli/#grafana-cli

**Comment by torkelo** [comment]
what is the purpose of this??

**Comment by aocenas** [comment]
@torkelo @marefr so what about this? I personally do not have a big need for this right now but I think @marefr mentioned another usecase where he would like to have it.

**Comment by marefr** [comment]
Have no strong opinions really, but since we have no strong use case for this now I say we close this for now until a user raises a problem.

**Pull request closed without merging**